javascript/javascript2/week1/script1.js

```diff
@@ -0,0 +1,26 @@
+const spiritAnimalNames = [ "Fish", "Fig","Ant", "Bear", "Frog", "Rabbit", "Giraffe", "Elephant", "Bat", "Polar"];
+                          
+var heading = document.createElement("h1");
+    heading.setAttribute("id", "heading");
+    document.body.appendChild(heading);
+    heading.innerText = "Spirit Animal Name Generator";
+
+var p = document.createElement("p");
+    p.setAttribute("class", "text");
+    document.body.appendChild(p);
+    p.innerText = "(Enter your name to get your Spirit Animal)";
+
+function spiritAnimalNameGenerator() {
+    
+    var input = document.getElementById("firstName").value;
+        if(input === "") {
+            alert("Please enter your first name");
+            return;
+        }
+        else {
+        let  randomIndex = Math.floor(Math.random() * 10);
```

Comment:
Nice usage of let. It is in general better to use `let` then `var` keyword to declare variables. In your code all of the variables could actually be declared with `const` as we do not reaasign any of them.